client/app/components/ServicesTemplate/data.json

```diff
@@ -6,48 +6,51 @@
           "id": "1",
           "title": "Name and Address",
           "content": "Verify telephone numbers, accounts, email addresses, names, SSN, and addresses",
-          "url": "telified",
+          "url": "3D",
           "img": "3D"
         },
         {
           "id": "2",
           "title": "Active / Inactive",
           "content": "Retrive additional data based on telephone, email, SSN, or address",
-          "url": "telified",
-          "img": "enrichment"
+          "url": "AI",
+          "img": "AI"
         },
         {
           "id": "3",
           "title": "Consent Re-verification",
           "content": "Use our best of breed fuzzy matching to compare multiple contact records",
-          "url": "telified",
-          "img": "comparison"
+          "url": "CR",
+          "img": "CR"
         },
         {
           "id": "4",
           "title": "Inter-Continental",
           "content": "CNAM, LIDB, Verification, and Status hosting services of the telecom industry",
-          "url": "telified",
-          "img": "telecom"
+          "url": "IC",
```

Comment:
lowercase only for naming files, while mac isn't case sensitive, Linux is, and it's pita to debug later